model/src/main/java/com/ge/predix/acs/rest/AttributeAdapterConnection.java

```diff
@@ -32,6 +36,7 @@ public AttributeAdapterConnection(final AttributeAdapterConnection other) {
         this.uaaClientSecret = other.uaaClientSecret;
     }
 
+    @ApiModelProperty(value = "URL to a remote attribute store adapter", required = true)
```

Comment:
Did we decide to change this or leave it? Latest rebase doesn't show this line being changed.